src/resources/parser/texture/hdr.js

```diff
@@ -19,78 +29,80 @@ Object.assign(Reader.prototype, {
             result += c;
         }
         return result;
-    },
+    }
 
-    readUint8: function () {
+    readUint8() {
         return this.view.getUint8(this.offset++);
-    },
+    }
 
-    readUint8s: function (result) {
+    readUint8s(result) {
         var view = this.view;
         for (var i = 0; i < result.length; ++i) {
             result[i] = view.getUint8(this.offset++);
         }
-    },
+    }
 
-    peekUint8s: function (result) {
+    peekUint8s(result) {
         var view = this.view;
         for (var i = 0; i < result.length; ++i) {
             result[i] = view.getUint8(this.offset + i);
         }
     }
-});
+}
 
 /**
  * @class
  * @name HdrParser
  * @implements {TextureParser}
  * @classdesc Texture parser for hdr files.
  */
-function HdrParser(registry) {
-    this.maxRetries = 0;
-}
+class HdrParser {
```

Comment:
And if this _is_ made public with documentation, currently, docs are pretty bare. 😄

Reply:
Basically, consider whether this should possibly be private.

Reply:
Private - and I'll do the same to the other parsers?

Reply:
Fine by me.